bmadx/structures.py

```diff
@@ -1,6 +1,21 @@
 from collections import namedtuple
+from bmadx.constants import M_ELECTRON
 
-Particle = namedtuple('Particle', 'x px y py z pz s p0c mc2')
+Particle = namedtuple(
```

Comment:
Is there a strong case where default values are required? Otherwise we shouldn't specify default values

Reply:
Usually, s=0 and mc2=M_ELECTRON, so that saves some time when testing with bmad-x

Reply:
I removed the defaults